client/my-sites/controller.js

```diff
@@ -112,46 +111,62 @@ function renderNoVisibleSites( context ) {
 	);
 }
 
-function isPathAllowedForDomainOnlySite( pathname, domainName ) {
+function renderSelectedSiteIsDomainOnly( { reactContext, selectedSite } ) {
+	const EmptyContentComponent = require( 'components/empty-content' );
+	const { store: reduxStore } = reactContext;
+
+	removeSidebar( reactContext );
+
+	renderWithReduxStore(
+		React.createElement( EmptyContentComponent, {
+			title: i18n.translate( 'This feature is not available for domains' ),
+			line: i18n.translate( 'To use this feature you need to create a site' ),
+			action: i18n.translate( 'Create New Site' ),
+			actionURL: '//dashboard.wordpress.com/wp-admin/index.php?page=my-blogs',
+			secondaryAction: i18n.translate( 'Domain Management' ),
```

Comment:
I agree that "Manage Domain" should be used here, even though it is similar to existing strings. Buttons in Calypso use active verbs in their labels. The string "Domain Management" suggested by @a8ci18n is not used in the same context. 

FWIW, a different existing string may be considered: "Manage Domain Settings". But this is longer than we would want, and including "Settings" in the label is too specific. I think that adding a new string in this case is justified.